src/utils/utils.js

```diff
@@ -15,3 +19,37 @@ export const setNativeValue = (element, value) => {
     valueSetter.call(element, value);
   }
 };
+
+export const getPxeBootPatch = vm => {
+  const patch = [];
+  const annotations = get(vm, 'metadata.annotations', {});
+  if (annotations[ANNOTATION_FIRST_BOOT]) {
+    if (annotations[ANNOTATION_FIRST_BOOT] === 'true') {
+      patch.push({
+        op: 'replace',
+        path: `/metadata/annotations/${ANNOTATION_FIRST_BOOT}`,
+        value: 'false',
+      });
+    } else {
+      // find bootable disk and change boot order
+      const bootableDiskIndex = getDisks(vm).findIndex(disk => disk.bootOrder === 2);
+      const bootableInterfaceIndex = getInterfaces(vm).findIndex(i => i.bootOrder === 1);
+
+      if (bootableDiskIndex !== -1 && bootableInterfaceIndex !== -1) {
+        patch.push(
+          {
+            op: 'replace',
+            path: `/spec/template/spec/domain/devices/disks/${bootableDiskIndex}/bootOrder`,
+            value: 1,
+          },
+          {
+            op: 'replace',
+            path: `/spec/template/spec/domain/devices/interfaces/${bootableInterfaceIndex}/bootOrder`,
+            value: 2,
+          }
+        );
+      }
+    }
+  }
+  return patch;
```

Comment:
dunno if `patches` would describes it better

Reply:
renamed to patches